app/models.py

```diff
@@ -534,6 +534,7 @@ class ServiceSmsSender(db.Model):
 
     id = db.Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
     sms_sender = db.Column(db.String(12), nullable=False)
+    sms_sender_service_id = db.Column(db.JSON())
```

Comment:
JSON is an unusual value for an ID.  I would expect this to be an integer, string, or UUID.  @k-macmillan says:

> The point of the field is to provide any additional information the service provider may want. In the case of twilio it's one string. In the future there may be others. To allow for that flexibility we're going with a json object.

Please document the basis for this decision in the code.  The class's docstring seems like a good place.

Reply:
Added documentation to explain reason for JSON.